Added support for Remo motion sensor

mlkh0225 opened the pull request:
Added support for Remo motion sensor and updated README

mlkh0225 committed:
Update index.js

Added support for Remo motion seneor

mlkh0225 committed:
Update README.md

Added description for motion sensor

hirosan18 reviewed (commented):
Thanks for the good PR!!

Is the 300,000 milliseconds for motion decisions related to the schedule parameter?
If the schedule parameters are changed, I don't think it will work as intended, do you have any ideas?

Also, can you add test? (You can write it in English)

Sorry for my bad English

mlkh0225 committed:
Update index.js

Adjusted for motion trigger to go off if motion detected since last refresh instead of fixed time (5 mins)

mlkh0225 commented:
Hi,

Yes, the threshold 300,000 ms is set base on the schedule, just changed it so sensor would be triggered if motion detected since last refresh.

mlkh0225 committed:
Update README.md

hirosan18 commented:
Can you add tests for added code?

mlkh0225 committed:
Merge remote-tracking branch 'upstream/master'

mlkh0225 committed:
Update index.js

Sync from Homebridge
Motion Sensor function added

mlkh0225 committed:
Motion option schema added
